.gitignore

```diff
@@ -102,3 +102,5 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+.vscode
```

Comment:
Needs `.sw*` for us Vim users, too.

Reply:
done.

Reply:
Shouldn't that be part of your overall .global_gitignore file?